OSDOCS-16940: DITA abstracts — OLM admin/user topics (#115359)
openshift/openshift-docs: modules/olm-changing-update-channel.adoc

[id="olm-changing-update-channel_{context}"]
= Changing the update channel for an Operator

[role="_abstract"]

> To move an installed Operator to a newer update channel, you can change the channel in the {product-title} web console. Selecting a new channel determines which Operator versions your subscription tracks and receives.

Reviewer: ```suggestion
To change the update channel for an installed Operator, you can use {product-title} web console. The update channel channel determines which Operator versions your subscription tracks and receives.
```
